README.md

```diff
@@ -14,23 +14,73 @@
   </a>
 </a>
 </p>
+
+<h2 align="center"><a href="https://killercoda.com/open-feature/scenario/flagd-demo" target="_blank">Try flagd in the Killercoda Playground (in browser)</a></h4>
 <!-- markdownlint-enable MD033 -->
 
 ## Features
 
+Flagd is a simple command line tool for fetching and evaluating feature flags for services.
+
+Think of it as a ready-made, open source, OpenFeature compliant feature flag backend system.
+
 - OpenFeature compliant with providers available in many languages
-- Multiple flag configuration sources including `files`, `http`, and `Kubernetes`
+- Multiple flag configuration sources including `files`, `http`, and `Kubernetes` can be used simultaneously
 - Accessible over gRPC and HTTP
 - Supports subscriptions to real-time flag change events
 - Flexible targeting rules based on [JSON Logic](https://jsonlogic.com/)
 - Lightweight daemon, with an emphasis on performance
 - Native support for metrics using Prometheus
 
-## Get started
+## Quick Start
 
-Flagd is a simple command line tool for fetching and evaluating feature flags for services.
-It is designed to conform with the OpenFeature specification.
-To get started, follow the installation instructions in the [docs](https://github.com/open-feature/flagd/blob/main/docs/README.md).
+![logical architecture of flagd](docs/images/flagd-logical-architecture.jpg)
+*High level, logical architecture of flagd*
+
+1. flagd can be run as a standalone-binary or container. [Download and install flagd or run it as a container](docs/usage/installation_options.md)
+
+2. Start flagd:
+
+    ```sh
+    // Start flagd
```

Comment:
Please make these separate commands so a user can use the copy to clipboard button in GitHub.